kitchen.yml

```diff
@@ -25,4 +29,11 @@ suites:
   - name: default
     run_list:
       - recipe[resolver-test::default]
-      - recipe[resolver::default]
+  - name: systemd_resolved
+    run_list:
+      - recipe[resolver-test::systemd_resolved]
+    includes:
+      - fedora-latest
```

Comment:
Would it make sense to include other platforms here or all systemd based ones?

Reply:
Good catch, IMO any platform version that uses systemd-resolvd should run this test

Reply:
It definitely would make sense but as of a cursory search I'm not sure of others that are using it at this point, Fedora only changed from nss-dns with 33 and with that being bleeding-edge I was thinking it's not widely used by default yet.

Reply:
I just confirmed Ubuntu 18.04 also uses systemd-resolved, so 20.04 probably does as well.

Reply:
Added